specs/deneb/fork-choice.md

```diff
@@ -104,8 +104,8 @@ def on_block(store: Store, signed_block: SignedBeaconBlock) -> None:
 
     # Add block timeliness to the store
     time_into_slot = (store.time - store.genesis_time) % SECONDS_PER_SLOT
-    is_before_attesting_interval = time_into_slot < SECONDS_PER_SLOT // INTERVALS_PER_SLOT
-    is_timely = get_current_slot(store) == block.slot and is_before_attesting_interval
+    is_before_late_block_cutoff = time_into_slot * 1000 < LATE_BLOCK_CUTOFF_MS
```

Comment:
if we're changing intervals to MS then it probably equally makes sense to be looking at `time_into_slot_ms`, maybe even `millis_per_slot` - maybe thats a bigger change than we'd want this PR though...

I'm not sure about the best name for `LATE_BLOCK_CUTOFF_MS` but its sounds a lot like late block reorg and we probably want to avoid that overlap given that's very different to whats happening here... maybe instead we should just be using `ATTESTATION_DUE_MS` here, and remove `LATE_BLOCK_CUTOFF_MS`.